shared/teams/invite-by-contact/fetch-contacts.native.tsx

```diff
@@ -0,0 +1,125 @@
+import * as Contacts from 'expo-contacts'
+import * as Container from '../../util/container'
+import * as React from 'react'
+import * as SettingsConstants from '../../constants/settings'
+import * as SettingsGen from '../../actions/settings-gen'
+import {ContactProps} from './index.native'
+import {e164ToDisplay} from '../../util/phone-numbers'
+import {NativeModules} from 'react-native'
+import logger from '../../logger'
+import TeamInviteByContact from './team-invite-by-contacts.native'
+
+// for sorting
+const strcmp = (a, b) => (a === b ? 0 : a > b ? 1 : -1)
+const compareContacts = (a: ContactProps, b: ContactProps): number => {
+  if (a.name === b.name) {
+    return strcmp(a.value, b.value)
+  }
+  return strcmp(a.name, b.name)
+}
+
+const fetchContacts = async (regionFromState: string): Promise<[Array<ContactProps>, string]> => {
```

Comment:
Would be nice to wrap this in a `memoize` from `util/memoize`

Reply:
wouldn't this result in having to kill the app and launch it again to see the contacts changing on this screen, after they were changed on the phone?

Reply:
Yes it would. Fair enough, don't need to add it then.